internal/app/machined/pkg/runtime/v1alpha1/platform/packet/packet.go

```diff
@@ -168,6 +165,10 @@ func (p *Packet) Configuration(ctx context.Context) ([]byte, error) {
 			DeviceCIDR:      fmt.Sprintf("%s/%d", addr.Address, addr.CIDR),
 			DeviceBond: &v1alpha1.Bond{
 				BondMode:       bondMode.String(),
+				BondDownDelay:  200,
+				BondMIIMon:     100,
+				BondUpDelay:    200,
+				BondHashPolicy: "layer3+4",
```

Comment:
I don't think this is safe to do across the board. Correct me if I am wrong @Ulexus, but I believe what we put here depends on how Equinix has things setup. And I don't think it is consistent for all machine types.

Reply:
there's nothing for it in Equinix metadata, so I must assume it's the "default" in whatever sense of this word

Reply:
Yeah, that was my fear. But I think we need to confirm this. There was something in the packet bonding setup that wasn't consistent. I don't remember what that was.

Reply:
It depends on what that bond mode is.  If it is failover, it needs no coordination.  If it is aggregated, it needs the switch's argeement.

Reply:
@Ulexus it's failover bond only (no link aggregation). 

if you look into #3007 - what I did was more or less copy-paste from what cloud-init does on Ubuntu box. As I understaqnd, hash policy shouldn't apply here, but :man_shrugging: , we can copy from what Packet does via cloud-init

Reply:
Okay, then it should be safe:  no coordination should be needed with Packet... and their thing is hard-coded anyway.  Perhaps they use that when they need to perform switch maintenance?  In any case, we should be safe here.